README.md

```diff
@@ -0,0 +1,176 @@
+# ROCM-CORE Introduction
+
+rocm-core is a utility which can be used to get ROCm release version. 
+It also provides the Lmod modules files for the ROCm release.
+getROCmVersion function provides the ROCm version. 
+
+Lmod module files can be loaded with the following commands.
+``` shell
+module load rocm/x.y or 
+module load rocm
+```
+
+You can find sources and binaries in our [GitHub repository](https://github.com/ROCm/rocm-core).
+
+> [!NOTE]
+> As with all ROCm projects, the documentation is open source. For more information, see [Contribute to ROCm documentation](https://rocm.docs.amd.com/en/latest/contribute/contributing.html).
+
+## Installing ROCM-CORE
+
+To install ROCM-CORE, you must first install these prerequisites:
+
+* A [ROCm](https://rocm.docs.amd.com/)-enabled platform
+
+### Installing with pre-built packages
+
+You can install ROCM-CORE on Ubuntu using
```

Comment:
you can install once the repo is set up.

Reply:
Updated description in the latest version.